eslint.config.mjs

```diff
@@ -0,0 +1,56 @@
+import eslint from "@eslint/js";
+import eslintConfigPrettier from "eslint-config-prettier";
+import eslintPluginPrettierRecommended from "eslint-plugin-prettier/recommended";
+import simpleImportSort from "eslint-plugin-simple-import-sort";
+import tseslint from "typescript-eslint";
+
+export default tseslint.config(
+  {
+    ignores: ["**/node_modules", "**/build", "**/coverage", "**/vendor"],
+  },
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  eslintConfigPrettier,
```

Comment:
[eslint-plugin-prettier docs](https://github.com/prettier/eslint-plugin-prettier) say that this is included, so should only need eslintPluginPrettierRecommended as the last item in the array.